httpclients/httpclient/src/main/java/com/okta/sdk/impl/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -76,66 +77,34 @@
 
     private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = Integer.MAX_VALUE/2;
     private static final String MAX_CONNECTIONS_PER_ROUTE_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.maxPerRoute";
-    private static final int MAX_CONNECTIONS_PER_ROUTE;
+    private static final int MAX_CONNECTIONS_PER_ROUTE = parseConfigValue(MAX_CONNECTIONS_PER_ROUTE_PROPERTY_KEY,
+                                                                          DEFAULT_MAX_CONNECTIONS_PER_ROUTE,
+                                                                         "Bad max connection per route value");
 
     private static final int DEFAULT_MAX_CONNECTIONS_TOTAL = Integer.MAX_VALUE;
     private static final String MAX_CONNECTIONS_TOTAL_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.maxTotal";
-    private static final int MAX_CONNECTIONS_TOTAL;
+    private static final int MAX_CONNECTIONS_TOTAL = parseConfigValue(MAX_CONNECTIONS_TOTAL_PROPERTY_KEY,
+                                                                      DEFAULT_MAX_CONNECTIONS_TOTAL,
+                                                              "Bad max connection total value");
 
-    private static final int DEFAULT_CONNECTION_VALIDATION_INACTIVITY = 5000; // 5sec
+    private static final int DEFAULT_CONNECTION_VALIDATION_INACTIVITY = 2000; // 2sec
```

Comment:
why this default changed?

Reply:
2s is the default of the underlying library, so I figured that would be a better starting point, I also think setting the max TTL will have bigger effect on limiting stale connections (so this now just exposes the configuration to the developer)